stdlib/public/core/CTypes.swift

```diff
@@ -219,6 +219,29 @@ extension UInt {
 }
 
 /// A wrapper around a C `va_list` pointer.
+#if arch(arm64) && os(Linux)
```

Comment:
This is wrong.  This applies to all AAPCS64 targets.  Please change this to:

```
#if arch(arm64) && !(os(macOS) || os(iOS) || os(tvOS) || os(watchOS) || os(Windows))
```

Reply:
I made a counter-suggestion below that we should just add a custom platform-condition check for the definition kind of `va_list`.

Reply:
No, this correct. The purpose of this PR is to add support for the aarch64/Linux platform not all AAPCS64 targets.